spec/services/aapb/batch_ingest/pbcore_xml_item_ingester_spec.rb

```diff
@@ -68,5 +68,41 @@
         expect(@essence_tracks.count).to eq 12
       end
     end
+
+    context 'given a PBCore Instantiation Document with Essence Tracks' do
+      # Before all, build a PBCore Instantiation Document with Essence Tracks
+      # Instantiations, and a Physical Instantiation and ingest it. Use the
+      # PBCore XML as the source data for a BatchItem instance, and then use the
+      # PBCoreXMLItemIngester to ingest the BatchItem. The return value is an
+      # Asset model instance on which we can write our expectations.
```

Comment:
@jasoncorum I think "the return value is an Asset model" should read "the return value is a DigitalInstantiation model", yes?